### DISCUSSION TITLE
Is it possible to use SpamScanner on top?

### DISCUSSION THREAD

**Discussion by WEBDIENSTE** [postalserver/postal#3179] [Feature suggestions]
https://github.com/spamscanner/spamscanner

**Comment by willpower232** [comment]
I've never heard of it so I'm going to presume Postal isn't compatible with it, unless it runs as a service Postal can use and that service inputs and outputs happens to be identical to something that is supported right now.

It looks like spamscanner uses clamav anyway so not sure what benefit you would get.
